config/v0.3.0/propagation.go

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package config // import "go.opentelemetry.io/contrib/config/v0.3.0"
+
+import (
+	"go.opentelemetry.io/contrib/propagators/autoprop"
+	"go.opentelemetry.io/otel/propagation"
+)
+
+func propagator(cfg configOptions) (propagation.TextMapPropagator, error) {
+	if cfg.opentelemetryConfig.Propagator == nil {
+		return autoprop.NewTextMapPropagator(), nil
+	}
+
+	names := make([]string, 0, len(cfg.opentelemetryConfig.Propagator.Composite))
+	for _, name := range cfg.opentelemetryConfig.Propagator.Composite {
+		if name != nil && *name != "" {
```

Comment:
I think that a `null` name in JSON and YAML are invalid and should result in returning an error.

I also do not see a reason to ignore empty names.

Reply:
Same as above: I think this makes it lenient for end-users (who are specifying it in a yaml file), but I'll follow the common practice for this repo.